### ISSUE TITLE
Warn or fail if attempting to write two different top level directories to the same output directory

### ISSUE THREAD

**Issue by natebosch** [dart-lang/build#2089]
`pbr build -o web:foo test:foo` will end up with just `web` output and `test` doesn't get output.

**Comment by natebosch** [comment]
Ah this is because the correct command is `pbr build -o web:foo -o test:foo` or `pbr build -o web:foo,test:foo`

Instead we maybe need to do better checking for rest args?

**Comment by jakemac53** [comment]
It should be sufficient to check that rest args actually correspond to a directory?

**Comment by natebosch** [comment]
Yeah I think if we are OK with the warning being fairly late we can check if it's a directory. We may not know the full set of top level directories until after the build.

**Comment by davidmorgan** [comment]
Closing as stale.
